File: src/main/java/com/pubnub/api/models/consumer/presence/PNGetStateResult.java
package com.pubnub.api.models.consumer.presence;

import com.google.gson.JsonElement;
import lombok.Builder;
import lombok.Getter;

import java.util.Map;

@Builder
@Getter
public class PNGetStateResult {

    private Map<String, JsonElement> stateByUUID;

}

File: pubnub-kotlin-core/src/main/kotlin/com/pubnub/api/models/consumer/presence/PNGetStateResult.kt
package com.pubnub.api.models.consumer.presence

import com.google.gson.JsonElement

/**
 * Result of the GetPresenceState operation.
 *
 * @property stateByUUID Map of UUIDs and the user states.
 */
class PNGetStateResult internal constructor(
    val stateByUUID: Map<String, JsonElement>
)
